examples/seismic/source.py

```diff
@@ -266,3 +266,34 @@ def wavelet(self, f0, t):
         tcut = 1.5 / agauss
         s = (t-tcut) * agauss
         return np.exp(-2*s**2) * np.cos(2 * np.pi * s)
+
+
+class DGaussSource(WaveletSource):
+    """
+    Symbolic object that encapsulate a set of sources with a
+    pre-defined 1st derivative wavelet of a Gaussian Source:
```

Comment:
these lines wrap earlier than the lines below, make it homogenous

Reply:
@FabioLuporini  I do not get what you mean here

Reply:
You mean like that for the note?
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html

Reply:
no I meant that these lines are how many, 60-65 chars long? while the paragraph below under Notes has lines which seem to be 75-80 chars

again, this is merely annoying nitpicking, sorry

Reply:
Fixed